libretro/libretro-core.c

```diff
@@ -4005,11 +4282,7 @@ static struct retro_disk_control_ext_callback diskControlExt = {
 
 static void fallback_log(enum retro_log_level level, const char *fmt, ...)
 {
-   (void)level;
-   va_list va;
-   va_start(va, fmt);
-   vfprintf(stderr, fmt, va);
-   va_end(va);
+   /* stub */

```

Comment:
Revert. Why are you removing this functionality?

Reply:
I didn't touch that code. Is that part of an upstream sync?